### PR TITLE
chore: update deno_core to 0.196.0

### PR DESCRIPTION
**Pull request by bartlomieju**

### CODE REVIEW

**Review by mmastrac — CHANGES_REQUESTED** [review]
Marking as request changes to prevent accidental merge

File: runtime/web_worker.rs [hunk 1]
```diff
@@ -528,7 +528,8 @@ impl WebWorker {
     };
 
     let bootstrap_fn_global = {
-      let context = js_runtime.global_context();
+      let realm = js_runtime.main_realm();
+      let context = realm.context();
```
Comment (andreubotella) [inline runtime/web_worker.rs:532]: `JsRuntime::global_context` was renamed to `main_context`, so you don't need to go through the realm.

**Review by mmastrac — COMMENTED** [review]
Looks good! Should we release deno_core and land?

**Comment by bartlomieju** [comment]
> Looks good! Should we release deno_core and land?

Pending https://github.com/denoland/deno_core/pull/74

**Comment by bartlomieju** [comment]
@mmastrac PTAL

**Review by mmastrac — APPROVED** [review]
LGTM

**Pull request merged**